### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by benja27**
-Add function to detect when user clicks on a checkbox
- Update the status of the checkbox on locastorage
- "clear completed" button function implemented
-No linter errors

### CODE REVIEW

File: src/modules.js/showDb.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+export const showDb = (db) => {
+  const container = document.getElementById('list-item-container');
+
+  container.innerHTML = '';
+
+  db.forEach((element, index) => {
```
Comment (AdnanOlarmmi) [inline src/modules.js/showDb.js:6]: - [ ] While you made great effort addressing and ensuring that the completion is controlled and saved to the local storage, kindly note that from the requirement is that clicking on the `clear completed` button should erase all the completed tasks in the array, meaning all the objects that has it's `completed` key value as `true`.  You can use `Array.filter()` higher order function for the functionality. 🙃

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @benja27, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

You're Almost there, funny one eyed alien

## Highlights
- Working app.
- Nice UI design.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules.js/showDb.js (hunk 1 above)
Comment (benja27) [reply src/modules.js/showDb.js:6]: fixed

**Review by DuaneDave — APPROVED** [review]
Hi @benja27 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ The UI looks good
✔️ Clearing completed todos is all functional

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**